examples/hypervisor_cluster_profile.pp

```diff
@@ -0,0 +1,79 @@
+################################################################################
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# NOTE: As with all resources, the found ensurable accepts a data as an optional filter field.
+
+# This created Hypervisor ClusterProfile if we use ServerProfileTemplate with OS DeploymentPlan
+oneview_hypervisor_cluster_profile{'hcp1 Create':
+    ensure => 'present',
+    data   => {
+      type                          => 'HypervisorClusterProfileV3',
+      name                          => 'Cluster5',
+      hypervisorManagerUri          => '/rest/hypervisor-managers/1ded903a-ac66-41cf-ba57-1b9ded9359b6',
+      path                          => 'DC2',
+      hypervisorType                => 'Vmware',
+      hypervisorHostProfileTemplate => {
+      serverProfileTemplateUri => '/rest/server-profile-templates/278cadfb-2e86-4a05-8932-972553518259',
+      hostprefix               => 'Test-Cluster-host'
+      }
+    }
+}
+
+# This creates Hypervisor Cluster Profile using Server Profile Template without OSDeploymentPlans
```

Comment:
i think you have interchange the comments here

Reply:
done